Cargo.toml

```diff
@@ -135,8 +135,8 @@ members = [
 	"cumulus/parachains/runtimes/test-utils",
 	"cumulus/parachains/runtimes/testing/penpal",
 	"cumulus/parachains/runtimes/testing/rococo-parachain",
-	"cumulus/polkadot-parachain",
-	"cumulus/polkadot-parachain/polkadot-parachain-lib",
+	"cumulus/polkadot-omni-node/polkadot-omni-node-lib",
+	"cumulus/polkadot-omni-node/polkadot-parachain",
```

Comment:
So just looking at this file, i can confirm that this is not what I intended (as noted, my original issue was not super clear). High level, expect to see something like: 

`cumulus/polkadot-parachain` should stay. This is also crucial for our internal devops to not put a bounty on my head. 
`cumulus/polkadot-omni-node` should be a new crate. 
`cumulus/polkadot-parachain-lib` ->  `cumulus/polkadot-omni-node/lib`

Reply:
Sorry, I misunderstood the requirement a bit. Will fix it.